### PR TITLE
Update copy of Users page

### PR DESCRIPTION
**Pull request by mendaomn**
## Description

This PR updates copy of Users page, and adds hardcoded strings to our localization file

## How Has This Been Tested?

- manually

## PR Quality Checklist

- [ ] I added tests to new or existing code
- [x] I removed hardcoded strings and used our `i18n` solution instead (see [here](https://github.com/aws-samples/pcluster-manager/pull/175/commits/fdc6b77987c87a26f51dbc8da5d371d95ef80601))
- [ ] I made sure no sensitive info gets logged at any time in the codebase (see [here](https://cheatsheetseries.owasp.org/cheatsheets/Logging_Cheat_Sheet.html)) (e.g. no user info or details, no stacktraces, etc.)
- [ ] I checked that infrastructure/update_infrastructure.sh runs without any error
- [x] I checked that `npm run build` builds without any error
- [x] I checked that clusters are listed correctly
- [ ] I checked that a new cluster can be created (config is produced and dry run passes)
- [ ] I checked that login and logout work as expected

In order to increase the likelihood of your contribution being accepted, please make sure you have read both the [Contributing Guidelines](../CONTRIBUTING.md) and the [Project Guidelines](../PROJECT_GUIDELINES.md)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: frontend/locales/en/strings.json [hunk 1]
```diff
@@ -583,6 +586,50 @@
       }
     }
   },
+  "users": {
+    "header": {
+      "title": "Users",
+      "description": "The list of users that have permission to access the AWS ParallelCluster User Interface."
+    },
+    "actions": {
+      "create": "Create user",
```
Comment (tmscarla) [inline frontend/locales/en/strings.json:595]: thought: we should rely on global actions where possible. If I am not mistaken `refresh` and `delete` should be already defined.

File: frontend/locales/en/strings.json [hunk 2]
```diff
@@ -583,6 +586,50 @@
...
+        }
+      },
+      "createForm": {
+        "phoneNumberPlaceholder": "+11234567890",
+        "emailPlaceholder": "email@domain.com"
+      },
+      "filtering": {
+        "loadingText": "Loading users...",
+        "countText": "Results: {{filteredItemsCount}}",
+        "filteringAriaLabel": "Enter search text",
```
Comment (tmscarla) [inline frontend/locales/en/strings.json:619]: praise: nice catch!

File: frontend/locales/en/strings.json [hunk 3]
```diff
@@ -583,6 +586,50 @@
...
+        "filteringPlaceholder": "Enter search text",
+        "empty": {
+          "title": "No users",
+          "subtitle": "No users to display."
+        },
+        "noMatch": {
+          "title": "No matches",
+          "subtitle": "No users match the filters.",
+          "action": "Clear filter"
```
Comment (tmscarla) [inline frontend/locales/en/strings.json:628]: thought: same as above. This action should be the same for all tables in which we use filtering.

**Review by tmscarla — APPROVED** [review]

File: frontend/locales/en/strings.json (hunk 1 above)
Comment (mendaomn) [reply frontend/locales/en/strings.json:595]: It's a good point. I thought about it and my conclusion was that we should be sure the two actions will always change copy "together". In general, re-using keys is a bit troublesome. We could maybe create a Component to re-use everywhere, thus we'll have only one key.

I propose we do it in future PRs

**Pull request merged**